app/models/cqm/measure.rb

```diff
@@ -57,10 +57,9 @@ class Measure
 
     # HQMF/Tacoma-specific Measure-logic related data
     field :population_criteria, type: Hash
-    field :data_criteria, type: Hash
-    field :source_data_criteria, type: Hash
     field :measure_period, type: Hash
     field :measure_attributes, type: Array
+    embeds_many :source_data_criteria, class_name: 'QDM::DataElement'
```

Comment:
With this change, should `embedded_in: :measure` be added to `app/models/qdm/basetypes/data_element.rb`?

Reply:
Yes, done.